lib/rust/parser/tests/parse.rs

```diff
@@ -471,6 +471,91 @@ fn import() {
     cases.into_iter().for_each(|(code, expected)| test(code, expected));
 }
 
+#[test]
+fn export() {
+    #[rustfmt::skip]
+    let cases = [
+        ("export Foo", block![(Export () () ((Ident export) (Ident Foo)) () ())]),
+        ("export Foo as Bar", block![
+            (Export () () ((Ident export) (Ident Foo)) ((Ident as) (Ident Bar)) ())]),
+        ("from Foo export Bar, Baz", block![
+            (Export
+             ((Ident from) (Ident Foo))
+             ()
+             ((Ident export) (OprApp (Ident Bar) (Ok ",") (Ident Baz)))
+             () ())]),
+        ("from Foo export all hiding Bar, Baz", block![
+            (Export
+             ((Ident from) (Ident Foo))
+             ()
+             ((Ident export) (Ident all))
+             ()
+             ((Ident hiding) (OprApp (Ident Bar) (Ok ",") (Ident Baz))))]),
+        ("from Foo as Bar export Baz, Quux", block![
```

Comment:
`from Foo as Bar export`!? Not sure what that can mean... why should I rename `Foo` to `Bar` when I export something else?

Reply:
I'm just supporting what is used in the libs. This pattern occurs in `distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Engine.enso`.